trfind/find.py

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env python
+
```

Comment:
not sure we need this shebang as the file doesn't do anything by itself, right?

Reply:
force of habit, removed